mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -62,3 +62,46 @@ data class ChessDocument(
     val whiteId: String? = null,
     val blackId: String? = null,
 )
+
+/**
+ * A document which represents a tournament of chess between many users. All the tournament
+ * documents are stored in the `/tournaments/` collection.
+ *
+ * @param uid the unique identifier for this tournament.
+ * @param adminId the unique identifier of the user administrating the tournament.
+ * @param name the name of the tournament.
+ * @param maxPlayers the maximum number of players than can join this tournament.
+ * @param bestOf the number of "best-of" rounds for the pool phase and the direct elimination phase.
+ * @param poolSize the target size of each pool. The number of pools derives from this number and
+ * the total number of players.
+ * @param eliminationRounds the number of direct elimination rounds. 1 for just a final, 2 for
+ * semi-finals, 3 for quarter-finals, etc...
+ * @param playerIds the [List] of unique identifier of users that have joined the tournament.
+ * @param pools the [List] of [PoolDocument]s keeping track of the pool phase of the tournament, if
+ * any.
+ */
+data class TournamentDocument(
+    @DocumentId val uid: String? = null,
+    val adminId: String? = null,
+    val name: String? = null,
+    val maxPlayers: Int? = null,
+    val bestOf: Int? = null,
+    val poolSize: Int? = null,
+    val eliminationRounds: Int? = null,
+    val playerIds: List<String>? = null,
+    val pools: List<PoolDocument>? = null,
+)
+
+/**
+ * A document which represents a pool in a tournament of chess. All the pool documents are stored
+ * inside their corresponding [TournamentDocument], in ``tournaments/tournamentId/`.
```

Comment:
```suggestion
 * inside their corresponding [TournamentDocument], in `tournaments/tournamentId/`.
```

Reply:
Fixed in https://github.com/epfl-SDP/android/pull/321/commits/d812f567f5999a8a38791dbbd9ee8f0b14205ce4